docs/source/peer-chaincode-devmode.rst

```diff
@@ -0,0 +1,129 @@
+Using dev mode
+==============
+
+Normally chaincodes are started and maintained by peer. However in “dev”
+mode, chaincode is built and started by the user. This mode is useful
+during chaincode development phase for rapid code/build/run/debug cycle
+turnaround.
+
+To keep this a realistic “dev” environment, we are going to keep it “out
```

Comment:
I'm not sure what this is trying to convey, but I don't think it adds anything to the doc and can be removed, i.e., what is "out-of-the-box" vs "not-out-of-the-box"

Reply:
done.